File: PPA/Assignment/Input_output/Input_output2/Program3.c
/*Program 3: What does the following C statement mean write detailed explanation?
scanf("%7s", str);   */



/*Ans) The statement is like controlling the input .Here it means it will only take maximum 7 charcters and if entered more it will only print 7 as demontrated in below program.*/



#include<stdio.h>

void main() {

	char ch[10];

	printf("Enter the string: ");
	scanf("%7s",ch);

	printf("%s\n",ch);
}

File: Java/Assignments/Assignment/Operators_1/Program3.java

class MixedIncDec {

	public static void main(String[] args) {

		int a= 10;
		int b= 20;
		int c= 50;
		int ans = 0;

		ans=a++ + ++b + ++c;

		System.out.println("Ans ="+ans);	//a++ -> 10 , ++b -> 21, ++c -> 51,  ans=10+21=31,  ans=31+51=82
		System.out.println("a ="+a);		//11
		System.out.println("b ="+b);		//21
		System.out.println("c ="+c);		//51
	}
}


